matches/matches.go

```diff
@@ -2,19 +2,161 @@ package main
 
 import (
 	"encoding/json"
+	"fmt"
 	"log"
 	"net/http"
+	"strconv"
 
+	"github.com/TempleEight/spec-golang/matches/dao"
+	valid "github.com/asaskevich/govalidator"
 	"github.com/gorilla/mux"
 )
 
 func main() {
 	r := mux.NewRouter()
-	r.HandleFunc("/matches", matchGetHandler).Methods(http.MethodGet)
+	r.HandleFunc("/matches/{id1}", matchListHandler).Methods(http.MethodGet)
+	r.HandleFunc("/match/{id}", matchGetHandler).Methods(http.MethodGet)
```

Comment:
Think these require a new route to be added to the Kong configuration since the url differs from the one previously defined